src/Stratis.Bitcoin.Tests/Utilities/NetworkExtensionsTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿using NBitcoin;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+
+namespace Stratis.Bitcoin.Tests.Utilities
+{
+    public class NetworkExtensionsTest
+    {
+        [Fact]
+        public void BitcoinNetworksAreBitcoin()
+        {
+            Assert.True(Networks.Networks.Bitcoin.Mainnet().IsBitcoin());
+            Assert.True(Networks.Networks.Bitcoin.Mainnet().IsBitcoin());
+            Assert.True(Networks.Networks.Bitcoin.Mainnet().IsBitcoin());
```

Comment:
I suppose these 3 lines are wrong, you want to test against Mainnet,testnet and regtest